rusk-wallet/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Change withdraw rewards to withdraw partial amount [#2538]
```

Comment:
CHANGELOG is not updated

Please rebase on top of current master and move this entry to the Unreleased section